### PR TITLE
docs: add DEM docs

### PR DESCRIPTION
**Pull request by shimoncohen**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✔                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

**Review by alebinson — DISMISSED** [review]
looks good

**Review by syncush — CHANGES_REQUESTED** [review]

File: docs/MapColonies/DEM/Services/catalog/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+id: dem-catalog-service
+slug: dem-catalog
+title: Catalog 🗃️
+tags:
+  - DEM
```
[the remark this answers is not in the document]
Comment (shimoncohen) [reply docs/MapColonies/DEM/Services/catalog/README.md:6]: Added tags.

File: docs/MapColonies/DEM/Services/catalog/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - catalog
+  - smart-reality
+  - csw
+  - API
+  - qmesh
+  - quantaized-mesh
```
[the remark this answers is not in the document]
Comment (shimoncohen) [reply docs/MapColonies/DEM/Services/catalog/README.md:12]: Removed tags.

File: docs/MapColonies/DEM/Services/catalog/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+id: dem-catalog-service
+slug: dem-catalog
+title: Catalog 🗃️
+tags:
+  - DEM
+  - catalog
+  - smart-reality
```
[the remark this answers is not in the document]
Comment (shimoncohen) [reply docs/MapColonies/DEM/Services/catalog/README.md:8]: Removed tag.

File: docs/MapColonies/DEM/Services/data_coverage_service.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+### GetCoverage data from catalog materials with best resolution (coverageId=DTMBestResolution)
+
+***Request example:***
+```
+<DEM-SUPERIOR-SERVICE_URL>/wcs/?service=WCS&version=2.0.1&coverageId=DTMBestResolution&request=GetCoverage&format=image/geotiff&subset=Long(33,35)&subset=Lat(29,33)
+```
+
+***Response:***
+Tiff will be received
+
```
[the remark this answers is not in the document]
Comment (shimoncohen) [reply docs/MapColonies/DEM/Services/data_coverage_service.md:24]: Removed file.

File: docs/MapColonies/DEM/Guides/DEM_getting_started.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    end
+
+    a1 -- filter --> B[STEP 1<br/> Query CSW catalog]
+    a2 -- token --> B
+    B -- xml --> C[STEP 2<br/>Get layer's URI +<br/> coverage_id]
+    C -- constructed URI with<br/> coverage_id --> D1[STEP 3.1<br/>GetCoverage]
+    C -- layer_WMTS_LAYER_URI --> D2[STEP 3.2 <br/>Optional<br/>Cesium visualization]
+```
+> :no_entry: **Authentication must be integrated in order to communicate with CSW server.**<br/>
+> **See the principles [here](/ogc-protocols/ogc-csw-auth.md)**
```
[the remark this answers is not in the document]
Comment (shimoncohen) [reply docs/MapColonies/DEM/Guides/DEM_getting_started.md:37]: Fixed.

File: docs/MapColonies/DEM/Services/map_server.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+Those are standard protocols for serving pre-rendered or run-time computed georeferenced map tiles or DEM data over the Internet.
+
+It provides facilities for searching and retrieving raster layers data with the `GetCapabilities` and `GetTile` operations, all other operations are invalid.
+For the full capabilities provided by the service see [GetCapabilities](#getcapabilities)
+
+:::info
+**Authentication must be integrated in order to communicate with Map and Catalog services.**<br/>
+**See the principles [here](/docs/MapColonies/authentication)**
+:::
+
```
[the remark this answers is not in the document]
Comment (shimoncohen) [reply docs/MapColonies/DEM/Services/map_server.md:24]: Removed file.

File: docs/MapColonies/DEM/README.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+id: DEM-overview
+slug: dem-overview
+title: DEM
+sidebar_position: 1
+tags:
+ - DEM
+ - overview
+---
+
+## DEM (DTM / DSM)  <!-- {docsify-ignore} -->
```
[the remark this answers is not in the document]
Comment (shimoncohen) [reply docs/MapColonies/DEM/README.md]: Removed.

File: docs/MapColonies/DEM/Services/map_server.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (shimoncohen) [reply docs/MapColonies/DEM/Services/map_server.md]: Removed file.

File: docs/MapColonies/DEM/README.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+* **Vector** DEM / Quanitized Mesh is a representation of a continuous surface consisting a mesh of triangular facets. The most common format for Vector DEM is **Triangulated Irregular Network (TIN)**, which contains irregularly distributed nodes and lines (vertices) with X,Y,Z coordinates arranged in a network of non-overlapping triangles. These are meshes that contain only vertices where they define meaningful change in surface height. Vector DEM tiles are rendered more efficiently for displaying it in a 3D view.
+<figure>
+    <img src={require("/img/terrain_tin.png").default} style={{"display": "block","marginLeft": "auto","marginRight": "auto","width":"90%"}} />
+</figure>
+
+DEM can be rendered as 2D (raster) or 3D maps:
+
+* In **2D** view, the heights can be rendered as colors (based on some coloring scheme) or displayed as contour lines with hill shades.
+
+* In **3D** view, the terrain surface is visualized in an oblique three-dimensional perspective, using raster tiles or TIN mesh tiles. It can be augmented with a satellite imagery to make it look realistic.
```
[the remark this answers is not in the document]
Comment (shimoncohen) [reply docs/MapColonies/DEM/README.md]: Added images.

**Review by syncush — APPROVED** [review]

**Pull request merged**